ui/raidboss/data/06-ew/raid/p4s.ts

```diff
@@ -27,7 +31,14 @@ export interface Data extends RaidbossData {
   wellShiftKnockback?: boolean;
   beloneCoilsTwo?: boolean;
   bloodrakeCounter?: number;
+  act?: string;
+  colorHeadmarkerIds?: number[];
```

Comment:
```suggestion
  colorHeadmarkerIds: number[];
```

You have to check this being undefined in a lot of places.  I think it'd be simpler to always initialize it.

If you add this above the `triggers: [` entry:
```typescript
  initData: () => {
    return {
      colorHeadmarkerIds: [],
    };
  },
```